src/main/java/com/google/sps/data/GovSiteDetails.java

```diff
@@ -6,20 +6,20 @@
 import com.google.gson.annotations.SerializedName;
 
 /** 
- * This class represents the site details provided by NSW GOV API for each site avaialble for fitering.
+ * This class represents the site details provided by NSW GOV API for each site avaialble for fitering. A PDS. 
```

Comment:
What does PDS mean?

Reply:
we talked about this on Monday - PDS is a passive data structure, a container of info to be accessed by other processes. A PDS has public data member, and no methods. Like a struct!